docs/reference/components/sinks.cue

```diff
@@ -306,6 +306,46 @@ components: sinks: [Name=string]: {
 		}
 
 		if sinks[Name].features.send != _|_ {
+			if sinks[Name].features.send.keepalive != _|_ {
+				keepalive: {
+					common:      false
+					description: "Configures the sink keepalive behavior."
+					required:    false
+					type: object: {
+						examples: []
+						options: {
+							time: {
+								common:      false
+								description: "The time a connection needs to be idle before sending keepalive probes."
+								required:    false
+								type: uint: {
+									default: sinks[Name].features.send.keepalive.time
+									unit:    "seconds"
+								}
+							}
+							interval: {
```

Comment:
Maybe we should omit the `interval` and `retries` docs until https://github.com/timberio/vector/issues/4957 is resolved.

Reply:
Ah yes, we should. I thought they were supported since I saw them in tests.

Reply:
Yes, I was misleading a bit there. I left a note on the `TcpKeepaliveConfig` struct.